adbmanager/adbmanager.go

```diff
@@ -56,6 +56,17 @@ func (model Model) InstallAPKCmd(pathToAPK string, commandOptions *command.Opts)
 
 // RunInstrumentedTestsCmd builds and returns a `Command` for running instrumented tests on an attached device or emulator.
 // The `Command` can than be run by the consumer without needing to know the implementation details.
+//
+// `additionalTestingOptions` is a list of arbitrary key value pairs to be passed to the test runner.
+//
+// Example:
+//
+// If a value of `{"KEY1", "value1", "KEY2", "value2"}` is passed to `additionalTestingOptions`,
+// then it will be passed to the `adb` command like so:
+//
+// adb shell am instrument -e "KEY1" "value1" "KEY2" "value2" [...]
+//
+// See `adb` documentation for more info: https://developer.android.com/studio/command-line/adb#am
 func (model Model) RunInstrumentedTestsCmd(
```

Comment:
NOTE: by using cmd.PrintableCommandArgs we could easily test the generated command